tensorboard/backend/application.py

```diff
@@ -364,8 +371,9 @@ def reload_multiplexer(multiplexer, path_to_run):
   tf.logging.info('TensorBoard done reloading. Load took %0.3f secs', duration)
 
 
-def start_reloading_multiplexer(multiplexer, path_to_run, load_interval):
-  """Starts a thread to automatically reload the given multiplexer.
+def start_reloading_multiplexer(multiplexer, path_to_run, load_interval,
+                                use_process=False):
```

Comment:
nit: Is this function used in other places? If not, instead of having two levels of optional parameter, can we make `use_process` a required param?

Reply:
Done